test/data/jquery-1.9.1.js

```diff
@@ -4328,7 +4328,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 		assert( function( div ) {
 
 			// Select is set to empty string on purpose
-			// This is to test IE's treatment of not explictly
+			// This is to test IE's treatment of not explicitly
```

Comment:
This is inside a previous release of jQuery used internally so we wouldn't want to change it.